test/index.test.js

```diff
@@ -1,18 +1,15 @@
-var tape = require('tape');
+var abase = require('../lib/index.js');
+var fs = require('fs');
 var Hapi = require('hapi');
+var tape = require('tape');
 var path = require('path');
-var fs = require('fs');
-
-fs.writeFileSync(path.join(__dirname, 'schema.json'), JSON.stringify({}), 'utf8');
-
-var abase = require('../lib/index.js');
 
 tape('check wrong path throws error', function (t) {
   var server = new Hapi.Server();
   server.connection({ port: 0 });
 
   var opts = {
-    wrong_path: ''
+    wrong: path.join(__dirname, 'fixtures', 'schema.json')
```

Comment:
This test fails because the key of the options object is wrong, rather than because the path is wrong. It's a bit confusing. Can we make this explicit in the title of the test? Can we also maybe just delete the `schema.json` file since it only seems to be 'used' here (even though its not used)?